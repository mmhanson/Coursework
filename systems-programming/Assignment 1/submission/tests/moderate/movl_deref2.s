main:
	movl	$1012, %eax
	movl	$1012, %ebx
	movl	$555, %ecx
	movl	%ecx, 0(%eax)
	movl	0(%ebx), %edx
	printr	%edx
	ret
