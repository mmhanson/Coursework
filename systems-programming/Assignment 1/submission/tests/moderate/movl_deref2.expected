555 (0x22b)
